### PR TITLE
Bump version to v0.6.18.

### PR DESCRIPTION
**Pull request by qingling128**

### CODE REVIEW

File: fluent-plugin-google-cloud.gemspec [hunk 1]
```diff
@@ -10,7 +10,7 @@ eos
   gem.homepage      =
     'https://github.com/GoogleCloudPlatform/fluent-plugin-google-cloud'
   gem.license       = 'Apache-2.0'
-  gem.version       = '0.6.17'
+  gem.version       = '0.6.18'
   gem.authors       = ['Todd Derr', 'Alex Robinson']
```
Comment (igorpeshansky) [inline fluent-plugin-google-cloud.gemspec]: We'd need to update this at some point, but let's do that as a separate PR.

**Comment by dhrupadb** [comment]
Merging this on @qingling128 's behalf.
